algorithms/closest_pair.py

```diff
@@ -84,7 +84,12 @@ def _distance_between(point_1, point_2):
 
```

Comment:
The range for `i` in the above is `len(potential_closest_split_pairs)-1)` because you want to compare the point with the next point in the list. If you iterate up to the last point, there's nothing to compare it wihth